README.md

```diff
@@ -1,2 +1,54 @@
-Chameleon System
-================
+<p align="center">
+    <a href="https://chameleonsystem.com" target="_blank">
+        <img src="https://demo.chameleon-system.de/chameleon/mediapool/8/ae/chameleon-logo_id3834.png">
```

Comment:
Is this URL stable in future?

Reply:
At least for now it's good enough - may change in the future.